ci/build-linux.sh

```diff
@@ -22,15 +22,35 @@ if [ "${TRAVIS_OS_NAME}" != "linux" ]; then
 fi
 
 readonly IMAGE="cached-${DISTRO}-${DISTRO_VERSION}"
-echo "IMAGE = ${IMAGE}"
-
 sudo docker build -t "${IMAGE}:tip" \
      --build-arg DISTRO_VERSION="${DISTRO_VERSION}" \
      --build-arg CXX="${CXX}" \
      --build-arg CC="${CC}" \
      --build-arg NCPU="${NCPU:-2}" \
      --build-arg BUILD_TYPE="${BUILD_TYPE:-Release}" \
      --build-arg CHECK_STYLE="${CHECK_STYLE:-}" \
+     --build-arg SCAN_BUILD="${SCAN_BUILD:-}" \
      --build-arg GENERATE_DOCS="${GENERATE_DOCS:-}" \
      --build-arg CMAKE_FLAGS="${CMAKE_FLAGS:-}" \
      -f "ci/Dockerfile.${DISTRO}" .
+
+if [ "${SCAN_BUILD:-}" = "yes" ]; then
+    sudo docker run --rm -it --volume $PWD:/d ${IMAGE}:tip \
+	 bash -c 'if [ ! -z "$(ls -1d /tmp/scan-build-* 2>/dev/null)" ]; then cp -r /tmp/scan-build-* /d/scan-build-output; fi'
+    if [ -r scan-build-output/index.html ]; then
```

Comment:
This should be indented at the level of the previous line, no?

Reply:
`bash` is an argument to the `sudo docker` command, I thought we were indenting those by 2?
Anyway, changed the code to make this more readable.

Reply:
Sorry, misread, you're right.